code_samples/sequelize/currency/dbInit.js

```diff
@@ -8,10 +8,10 @@ const sequelize = new Sequelize('database', 'username', 'password', {
 });
 
 const CurrencyShop = sequelize.import('models/CurrencyShop');
-sequelize.import('models/Users')
+sequelize.import('models/Users');
 sequelize.import('models/UserItems');
 
-sequelize.sync().then(async () => {
+sequelize.sync().then(async() => {
```

Comment:
Same as above; the space should be there.

Reply:
Same as above for this as well, eslint file dictated it.